probe-rs/src/core/mod.rs

```diff
@@ -400,60 +399,59 @@ impl<'probe> Core<'probe> {
         self.inner.registers()
     }
 
+    /// Find the ID of the next available HW breakpoint.
```

Comment:
I find this a bit unclear, the breakpoints don't really have IDs. I would prefer something like `find_free_breakpoint_unit_index`, or similar.

Reply:
Renamed it to `find_free_breakpoint_comparator_index`. It is a bit verbose, but seldom used, and consistent with the STM32 docs